tests/object/stage/__snapshots__/test_stage.ambr

```diff
@@ -23,6 +23,45 @@
   
   '''
 # ---
+# name: test_execute_continue_on_error
+  '''
+  exe/s1.sql - SUCCESS
+  exe/s2.sql - FAILURE
+  exe/s3.sql - SUCCESS
+  +------------------------------+
+  | File       | Status  | Error |
+  |------------+---------+-------|
+  | exe/s1.sql | SUCCESS | None  |
+  | exe/s2.sql | FAILURE | Error |
+  | exe/s3.sql | SUCCESS | None  |
+  +------------------------------+
```

Comment:
I'm wondering if we really want to show duplicated info, won't table be enough?

Reply:
Messages are printed immediately after receiving the result, so user can see the progress. `--silent` flag hides messages. WDYT?

Reply:
Only if it is the same query (`select * from table`), in this case is many queries, so can't stream table. This is how sql command works. If we really want to stream table we should create separate ticket for it.

Reply:
Let's create the ticket, having option to stream to a table from generic generator not only cursor will be useful

Reply:
Created